README.md

```diff
@@ -286,6 +301,8 @@ installed regardless of your OS:
   2.8.12 or later.
 - [Python 3](http://www.python.org/): for utility scripts and running the test
 suite.
+- [Bazel](https://baze.build/) (optional): if building the source with Bazel,
+you need to install Bazel on your machine.
```

Comment:
Is there a particular version requirement?

Reply:
Done